.github/workflows/ci.yaml

```diff
@@ -71,17 +71,16 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@master
-    - uses: actions-rs/toolchain@v1
-      with:
-        toolchain: nightly
-        components: rustfmt, clippy
-        override: true
+
+    - name: nightly setup
+      run: rustup toolchain install nightly
+      run: rustup default nightly
```

Comment:
duplicate `run` key doesn't look right?

Reply:
No this is right, we could also do `cargo +nightly <command>` for the commands below.

Reply:
sorry I sholuld've been clearer, the command looks right, but the duplicate `run` keys are not valid YAML

Reply:
Oh! Huh, I wonder why this worked at all then.

Reply:
i dont think it did 😝 https://github.com/http-rs/surf/actions/runs/346519413

Reply:
It's because it ran no jobs, i.e. failed entirely.